test/models/claim_test.rb

```diff
@@ -107,10 +107,10 @@ def moderated_claim
   # Needing so many collaboratiors suggests the need for
   # service objects like Claim::Approval and Claim::Denial
   test 'user cannot moderate their own claim unless admin' do
-    skip """
+    skip '''
```

Comment:
Combine '' and "\n      Expecting this to fail because we have not yet implemented\n      roles like :admin that would help this pass.\n    " into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.<br>Combine "\n      Expecting this to fail because we have not yet implemented\n      roles like :admin that would help this pass.\n    " and '' into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.